### ISSUE TITLE
WooCommerce 7.6.0 flagged in Windows Defender Antivirus security intelligence

### ISSUE THREAD

**Issue by johnrom** [woothemes/woocommerce#37892]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

`woocommerce.7.6.0.zip` is flagged with latest definitions as of 

```
Security Intelligence Version 1.387.1679.0 4/20/2023 7:32 AM.
Detected: Trojan:Win32/Emali.A!cl
Details: This program is dangerous and executes commands from an attacker.
```

related: #37572 

### Expected behavior

Installing should work.

### Actual behavior

Installing does not work as the zip file is quarantined.

#### Reproduce in Chrome

Download: https://downloads.wordpress.org/plugin/woocommerce.7.6.0.zip

#### Reproduce with Composer

Create `composer.json`:

```json
{
    "php": ">=8.0",
    "wpackagist-plugin/woocommerce": "7.5.0"
}
```

And run:

```sh
composer install
```

### WordPress Environment

Cannot install in order to get status report.

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by wizzyto12** [comment]
I can reproduce as well on all our office PC's.

**Comment by gasparapsag** [comment]
hi there, same here I can´t install the new woocommerce as Google Chrome stops the download and if I force it by any other way Windows defender block it too

**Comment by STV11C** [comment]
I also have the same problem with Google Chrome stopping the download.
<img width="170" alt="Screenshot" src="https://user-images.githubusercontent.com/5532744/233776043-e50bf60b-1b65-4001-9971-f250422de83e.png">

**Comment by tammullen** [comment]
Thanks for flagging this

We have reported this issue as a false positive to Microsoft and hopefully they will update their definitions.

**Comment by tammullen** [comment]
Update: definitions have been updated.

If you are still seeing it flagged please update your Windows Defender Definitions https://www.microsoft.com/en-us/wdsi/defenderupdates and try scanning again.

**Comment by davidzonagrafica** [comment]
It's working!

Updating definitions, downloading the WooCommerce 7.6.0 ZIP and uncompress on Windows 10 no longer is detected as Troyan. 

**STEPS FOR UPDATING MICROSOFT DEFENDER:**

Open cmd as Administrator

````
cd %ProgramFiles%\Windows Defender
MpCmdRun.exe -removedefinitions -dynamicsignatures
MpCmdRun.exe -SignatureUpdate
````

**Comment by johnrom** [comment]
Confirmed, thanks!

**Comment by davidzonagrafica** [comment]
Same problem today, false positive again :(

![2023-05-17--134940--Seguridad de Windows](https://github.com/woocommerce/woocommerce/assets/91727708/5838bf43-f407-4e04-94f2-81a2c1dde97a)

**Comment by koossaayy** [comment]
Same problem for today also

**Comment by tammullen** [comment]
Thanks for flagging this

We have reported this issue with 7.7.0 as a false positive to Microsoft and hopefully they will update their definitions as soon as possible.

**Comment by tammullen** [comment]
Update: definitions have been updated again we believe.

If you are still seeing it flagged please update your Windows Defender Definitions https://www.microsoft.com/en-us/wdsi/defenderupdates and try scanning again.

**Comment by rrennick** [comment]
It looks like this has been addressed for 7.7.0.
